src/pages/TaskList/TaskList.js

```diff
@@ -0,0 +1,46 @@
+import {Link} from 'react-router-dom';
+import {Tabs, Tab} from '../../components/Tabs/index';
+import { getTasks} from "../../services/tasks";
+
+export class TaskList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tasksInWeek: []
+    };
+  }
+  days = ['Вс', 'Пн', 'Вт', 'Ср', 'Чт', 'Пт', 'Сб'];
```

Comment:
in Task.js you have english days ` const days = ['Sun', 'Mon', 'Thur', 'Wed', 'Th', 'Fr', 'St'];`
Be consistent. Use constants)